### DISCUSSION TITLE
Cache file deletion error

### DISCUSSION THREAD

**Discussion by sypro2020** [filecoin-project/community#66] [General]
An error occurred in my lotus program. I did 7 seal tasks at the same time. After all seal tasks are completed, the cache data of two tasks is not completely deleted from PC1 Worker /cache. The 32G cache file of same two tasks in PC1 worker /cache/sealed was also not deleted correctly.
![1371597187289__pic](https://user-images.githubusercontent.com/67464208/89959778-89215400-dc3d-11ea-888b-51fdf0ba6cad.jpg)
![1381597189301__pic](https://user-images.githubusercontent.com/67464208/89959798-90486200-dc3d-11ea-989d-e0adbb09358c.jpg)

**Comment by BBQFIL** [comment]
me too

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
if the sectors is sealed and in store storage you can delete them.

do this: move them to another place, outside of a seal storage. wait for a wdPost to pass. if the wdPost the sectors should be proven in passes without the sector getting faulty - delete the files
